src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_flexible_commands.py

```diff
@@ -467,3 +469,369 @@ def _test_flexible_server_replica_mgmt(self, database_engine, resource_group):
+                                 .format(database_engine, resource_group_2, servers[0])).get_output_in_json()
+
+        show_result_2 = self.cmd('{} flexible-server show -g {} -n {}'
+                                 .format(database_engine, resource_group_2, servers[1])).get_output_in_json()
+
+        self.assertEqual(show_result_1['delegatedSubnetArguments']['subnetArmResourceId'],
+                         '/subscriptions/{}/resourceGroups/{}/providers/Microsoft.Network/virtualNetworks/{}/subnets/{}'.format(
+                             self.get_subscription_id(), resource_group_1, vnet_name, subnet_name))
+
+        self.assertEqual(show_result_2['delegatedSubnetArguments']['subnetArmResourceId'],
+                         '/subscriptions/{}/resourceGroups/{}/providers/Microsoft.Network/virtualNetworks/{}/subnets/{}'.format(
+                             self.get_subscription_id(), resource_group_2, vnet_name_2, 'Subnet' + servers[1][6:]))
+
+        # delete all servers
+        self.cmd('{} flexible-server delete -g {} -n {} --yes'.format(database_engine, resource_group_2, servers[0]),
+                 checks=NoneCheck())
+
+        self.cmd('{} flexible-server delete -g {} -n {} --yes'.format(database_engine, resource_group_2, servers[1]),
+                 checks=NoneCheck())
+
+        time.sleep(15 * 60)
+
+        # remove delegations from all vnets
+        self.cmd('network vnet subnet update -g {} --name {} --vnet-name {} --remove delegations'.format(resource_group_1,
+                                                                                                         subnet_name,
+                                                                                                         vnet_name))
+
+        self.cmd('network vnet subnet update -g {} --name {} --vnet-name {} --remove delegations'.format(resource_group_2,
+                                                                                                         'Subnet' +
+                                                                                                         servers[1][6:],
+                                                                                                         vnet_name_2))
+
+        # remove all vnets
+        self.cmd('network vnet delete -g {} -n {}'.format(resource_group_1, vnet_name))
+        self.cmd('network vnet delete -g {} -n {}'.format(resource_group_2, vnet_name_2))
+
+
+class FlexibleServerPublicAccessMgmtScenarioTest(ScenarioTest):
+    postgres_location = 'eastus'
+    mysql_location = 'westus2'
+
+    @AllowLargeResponse()
+    @ResourceGroupPreparer(location=postgres_location)
+    @live_only()
+    def test_postgres_flexible_server_public_access_mgmt(self, resource_group):
+        self._test_flexible_server_public_access_mgmt('postgres', resource_group)
+
+    @AllowLargeResponse()
+    @ResourceGroupPreparer(location=mysql_location)
+    @live_only()
```

Comment:
May I know the reason to make these two tests live only?

Reply:
This is because the firewall name consists of the timestamp( to keep it unique). Recording the test with non-deterministic value like the timestamp would fail the test always.